test/ReachSets/unit_discretization.jl

```diff
@@ -4,92 +4,99 @@ let # preventing global scope
 
     # ===============================================================
     # Discretization of a continuous-time system with constant input
     # ===============================================================
-    U = Ball2(ones(4), 0.5)
-    cont_sys = ContinuousSystem(A, X0, U)
+    U = ConstantInput(Ball2(ones(4), 0.5))
+    B = Matrix{Float64}(1.0I, 4, 4)
+    cont_sys = IVP(CLCCS(A, B, nothing, U), X0)
 
-    # no bloating, do not use Pade approximation
-    discr_sys = discretize(cont_sys, δ, approx_model="nobloating", pade_expm=false)
+    # no bloating
+    discr_sys = discretize(cont_sys, δ, approximation="nobloating")
     @test inputdim(discr_sys) == 4
-    #@test length(discr_sys.U) == 1 # a constant input has no length method!
-    inputs = next_set(inputset(discr_sys)) # getter: inputset(discr_sys) = discr_sys.s.U
+
+    inputs = next_set(inputset(discr_sys))
     @test dim(inputs) == 4
-    @test isa(inputs, LinearMap)
+    @test isa(inputs, LazySets.LinearMap)
     @test isa(inputs.X, Ball2) && inputs.X.center == ones(4) && inputs.X.radius == 0.5
 
     # no bloating, use Pade approximation
-    discr_sys = discretize(cont_sys, δ, approx_model="nobloating", pade_expm=true)
+    discr_sys = discretize(cont_sys, δ, approximation="nobloating", exp_method="pade")
     @test inputdim(discr_sys) == 4
 
-    # bloating, do not use Pade approximation
-    discr_sys = discretize(cont_sys, δ, pade_expm=false)
+    # bloating, use scaling and squaring method
+    discr_sys = discretize(cont_sys, δ, exp_method="base")
     @test inputdim(discr_sys) == 4
 
-    #@test length(discr_sys.U) == 1
     inputs = next_set(inputset(discr_sys))
     @test dim(inputs) == 4
     @test isa(inputs, MinkowskiSum)
 
     # bloating, use Pade approximation
-    discr_sys = discretize(cont_sys, δ, pade_expm=true)
+    discr_sys = discretize(cont_sys, δ, exp_method="pade")
     @test inputdim(discr_sys) == 4
 
-    discr_sys = discretize(cont_sys, δ, approx_model="firstorder")
+    discr_sys = discretize(cont_sys, δ, approximation="firstorder")
     @test inputdim(discr_sys) == 4
 
     # ===================================================================
     # Discretization of a continuous-time system with time-varying input
     # ===================================================================
     Ui = [Ball2(0.01*i*ones(4), i*0.2) for i in 1:3]
-    cont_sys = ContinuousSystem(A, X0, Ui)
+    cont_sys = IVP(CLCCS(A, Matrix(1.0I, 4, 4), nothing, VaryingInput(Ui)), X0)
 
-    # no bloating, do not use Pade approximation
-    discr_sys = discretize(cont_sys, δ, approx_model="nobloating", pade_expm=false)
+    # no bloating
+    discr_sys = discretize(cont_sys, δ, approximation="nobloating")
     Ui_d = inputset(discr_sys)
     @test length(Ui_d) == 3
     for (i, inputs) in enumerate(discr_sys.s.U)
         if i == 1
             @test dim(inputs) == 4
-            @test isa(inputs, LinearMap)
+            @test isa(inputs, LazySets.LinearMap)
```

Comment:
Maybe add `using LazySets: LinearMap`.
In fact, we already have this line in `runtests.jl`.